config/routes.rb

```diff
@@ -5,7 +5,9 @@
   devise_for :student, :controllers => { :invitations => 'invitations', :registrations => 'registrations' }
   devise_for :admins, skip: :registrations
 
-  resources :students, only: [:update]
+  resources :students, only: [:update] do
+    get 'internships', to: 'students#internships'
```

Comment:
ooh, let's not use a non-restful route here. change to:

```
resources :students, only: [:update] do
  resources :internships, only: [:show]
end
```

update the controller and template filename accordingly.